general-knowledge/about-mycrypto/system-mapping.md

```diff
@@ -0,0 +1,333 @@
+---
+title: System Mapping

```

Comment:
The title and description are unclear to me. I don't see the relation between this title and the content.

Reply:
What would you suggest? The idea is to map systems with their functions and technology behind it